docs/ProfilingNonJavaApplications.md

```diff
@@ -21,10 +27,38 @@ typedef const char* (*asprof_error_str_t)(asprof_error_t err);
 DLLEXPORT asprof_error_t asprof_execute(const char* command, asprof_writer_t output_callback);
 typedef asprof_error_t (*asprof_execute_t)(const char* command, asprof_writer_t output_callback);
 ```
+To use it in a C/C++ application, include asprof.h. Below is an example usage showing how to use async-profiler command with the API. The :
+```
+int main() {
+    void* lib = dlopen(path/to/libasyncProfiler.so", RTLD_NOW);
+    if (lib == NULL) {
+        fail("Failed to load libasyncProfiler.so");
+    }
+
+    asprof_init_t asprof_init = dlsym(lib, "asprof_init");
```

Comment:
null check.

Reply:
Sure will add it.

Reply:
Addressed in commit [c265f59](https://github.com/async-profiler/async-profiler/pull/1029/commits/c265f5915211c7fb2d3c7e292883f1d85f85a86d)